pkl/evaluator_exec.go

```diff
@@ -54,6 +54,13 @@ func NewProjectEvaluator(ctx context.Context, projectDir string, opts ...func(op
 // because it lessens the overhead of each successive evaluator.
 func NewProjectEvaluatorWithCommand(ctx context.Context, projectDir string, pklCmd []string, opts ...func(options *EvaluatorOptions)) (Evaluator, error) {
 	manager := NewEvaluatorManagerWithCommand(pklCmd)
+	managerClosed := false
+	defer func() {
+		if !managerClosed {
+			manager.Close()
+		}
+	}()
+
```

Comment:
These return an evaluator, where, once the evaluator is closed, the manager is closed too. I don't think there's any leak here.